keps/693-multikueue/README.md

```diff
@@ -316,6 +323,76 @@ that ware created by this multikueue manager cluster and delete them if their lo
 - `WorkerLostTimeout` - defines the time a local workload's multikueue admission check state is kept Ready
 if the connection with its reserving worker cluster is lost.
 
+
+### MultiKueue Dispatcher API
+
+The MultiKueue Workload Controller consists of nomination nd synchronization: 
+first, it nominates worker clusters for each workload based on the configured dispatching mode;
+then, it synchronizes the workload by creating and managing its copies in the nominated clusters. 
+This ensures that only one cluster ultimately admits and runs the workload, 
+while the management cluster maintains up-to-date status by tracking the admitted job.
+
+#### Cluster Nomination
+Cluster nomination phase looks differently depending on the configured dispatching mode.
+MultiKueue supports three dispatching modes: AllAtOnce (`kueue.x-k8s.io/multikueue-dispatcher-all-at-once`),
+Incremental (`kueue.x-k8s.io/multikueue-dispatcher-incremental`), and External.
+
+* **AllAtOnce Mode**:  
+  In this mode, all available clusters are considered for dispatching at once, and the workload is copied to each worker cluster.
+
+* **Incremental Mode**:  
+  In this mode, clusters are nominated incrementally in rounds of fixed duration (5 minutes per round). 
+  The process begins by nominating an initial set of 3 clusters, which are added to the `.spec.multiKueueNominatedClusters` field.
+  During the first 5-minute round, only these clusters are eligible to admit the workload.
+  If none of the initial clusters admit the workload within the round's duration, the next round begins, and 3 additional clusters are nominated.
+  This incremental approach continues, adding 3 more clusters in each subsequent round, until the workload is admitted or all eligible clusters have been considered.
+  This strategy allows for a controlled and gradual expansion of candidate clusters, rather than dispatching the workload to all clusters at once,
+  which can help optimize resource usage and reduce unnecessary workload duplication.
```

Comment:
```suggestion
  If none of the initial clusters admit the workload within the round's duration, the next round begins, and 3 additional clusters are nominated.
  This incremental approach continues, adding 3 more clusters in each subsequent round, until the workload is admitted or all eligible clusters have been considered.
  This strategy allows for a controlled and gradual expansion of candidate clusters, rather than dispatching the workload to all clusters at once.
```

Reply:
The last sentence "  which can help optimize resource usage and reduce unnecessary workload duplication." is motivation rather than "Design detail". Motivation is better to capture in Story section.